PCbuild/_freeze_module.vcxproj

```diff
@@ -394,7 +394,7 @@
+	<Exec Command='$(PythonForBuild) "$(PySourcePath)Tools\scripts\deepfreeze.py" "$(PySourcePath)Python\frozen_modules\importlib._bootstrap.h:importlib._bootstrap" "$(PySourcePath)Python\frozen_modules\importlib._bootstrap_external.h:importlib._bootstrap_external" "$(PySourcePath)Python\frozen_modules\zipimport.h:zipimport" "$(PySourcePath)Python\frozen_modules\abc.h:abc" "$(PySourcePath)Python\frozen_modules\codecs.h:codecs" "$(PySourcePath)Python\frozen_modules\io.h:io" "$(PySourcePath)Python\frozen_modules\_collections_abc.h:_collections_abc" "$(PySourcePath)Python\frozen_modules\_sitebuiltins.h:_sitebuiltins" "$(PySourcePath)Python\frozen_modules\genericpath.h:genericpath" "$(PySourcePath)Python\frozen_modules\ntpath.h:ntpath" "$(PySourcePath)Python\frozen_modules\posixpath.h:posixpath" "$(PySourcePath)Python\frozen_modules\os.h:os" "$(PySourcePath)Python\frozen_modules\site.h:site" "$(PySourcePath)Python\frozen_modules\stat.h:stat" "$(PySourcePath)Python\frozen_modules\importlib.util.h:importlib.util" "$(PySourcePath)Python\frozen_modules\importlib.machinery.h:importlib.machinery" "$(PySourcePath)Python\frozen_modules\runpy.h:runpy" "$(PySourcePath)Python\frozen_modules\__hello__.h:__hello__" "$(PySourcePath)Python\frozen_modules\__phello__.h:__phello__" "$(PySourcePath)Python\frozen_modules\__phello__.ham.h:__phello__.ham" "$(PySourcePath)Python\frozen_modules\__phello__.ham.eggs.h:__phello__.ham.eggs" "$(PySourcePath)Python\frozen_modules\__phello__.spam.h:__phello__.spam" "$(PySourcePath)Python\frozen_modules\frozen_only.h:frozen_only" "-o" "$(PySourcePath)Python\deepfreeze\deepfreeze.c"'/>
```

Comment:
I'd still like to see this long line folded too...

Reply:
Done

Reply:
I have split on multiple lines now.